### PR TITLE
finished the code challenge

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401-challenges/src/main/java/code401/challenges/RepeatedWord/RepeatedWord.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package code401.challenges.RepeatedWord;
+
+import java.util.HashMap;
+
+public class RepeatedWord {
+
+    public String occurMoreThanOnce(String sentence){
+        String[] wordsArr = sentence.split("\\W+");
+        HashMap<String, Integer> occuranceMap = new HashMap<>();
+        int dummyCnt = 0;
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/RepeatedWord/RepeatedWord.java:10]: Instead of creating a counter you could have just inserted any value into the hashmap since the value didn't matter
